lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -64,6 +69,19 @@ private void Run()
                 .Set(DriverConfiguration.CustomTraceLevel, Level.Verbose.ToString())
                 .Build();
 
+            if (includeContainerConfiguration)
+            {
+                ICsConfigurationBuilder containerConfigurationBuilder = TangFactory.GetTang().NewConfigurationBuilder()
+                    .BindImplementation(typeof(ITcpPortProvider), typeof(SetTcpPortProvider));
+
+                foreach (int port in ports)
+                {
+                    containerConfigurationBuilder.BindSetEntry<TcpPortSet, int>(port.ToString());
+                }
+                IConfiguration containerConfiguration = containerConfigurationBuilder.Build();
+                helloDriverConfiguration = Configurations.Merge(containerConfiguration, helloDriverConfiguration);
+            }
+
```

Comment:
We can't have this logic specific to HelloReef as it's internal functionality and needs to be transparent to the end user.